plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/serialize/SerializerDescriptor.java

```diff
@@ -41,7 +41,7 @@ public class SerializerDescriptor extends AbstractDescriptor
 
         this.id = config.getAttribute("id");
         this.serializerType = new ObjectType(config.getAttribute("class"));
-        this.distributed = CommonUtils.getBoolean(config.getAttribute("distributed"), true);
+        this.distributed = CommonUtils.getBoolean(config.getAttribute("supportsDistributedMode"), true);
```

Comment:
Please use `RegistryConstants#ATTR_SUPPORTS_DISTRIBUTED_MODE`

Reply:
there is no dependence on `org.jkiss.dbeaver.registry`

Reply:
Oh. Okay